guides/09_Amazon_Voice_Focus.md

```diff
@@ -283,9 +283,27 @@ A specification is used to derive a runtime _configuration_ when a transformer i
 
 ## Enabling Voice Focus with Echo Reduction
 
-Enabling Voice Focus with Echo Reduction is a two step process: 
+Enabling Voice Focus with Echo Reduction is a three step process: 
 
-### 1. Creating a meeting with support for Echo Reduction
+### 1. Changing the namespace to Amazon Chime Meetings SDK
+
+To create meetings, the following line of code is used to address the namespace:
+
+```typescript
```

Comment:
Since you are not using typescript, we can use just `js` or `javascript` I think.

Reply:
May be you can update the rest of the doc if not using types, but up to you if you want to change or you want to keep it.